Flowpack/media-ui
# WIP FEATURE: Store selected tag and collection

**What I did**

Store last selected tag and collection in users browser session. Resolves #7 

**How I did it**

Last selected collection and tag are persisted to local storage the same way as the selected asset source and view mode.

**How to verify it**

Select tag & collection and switch between backend module and ui selection mode, or reload browser window. The selected should stay the same.
